server/src/main/java/org/opentosca/toscana/plugins/cloudformation/CloudFormationLifecycle.java

```diff
@@ -74,9 +86,16 @@ public void transform() {
 
             fileAccess.access("output/template.yaml").appendln(cfnModule.toString()).close();
             logger.info("Transformation to CloudFormation successful.");
-        } catch (Exception e) {
+        } catch (IOException ie) {
+            logger.error("File access error");
+            throw new TransformationFailureException("Could not write with fileAccess", ie);
+        } catch (TransformationFailureException tfe) {
             logger.error("Transformation to CloudFormation unsuccessful. Please check the StackTrace for more Info.");
-            e.printStackTrace();
+            tfe.printStackTrace();
```

Comment:
there is no need to print the stacktrace, because the ExecutionTask (the part of the core that handles the excution of transformation)